nursery_store.go

```diff
@@ -114,15 +114,15 @@ type NurseryStore interface {
 	// executed after receiving confirmation of the preschool output.
 	// Incoming HTLC's we need to go to the second-layer to claim, and also
 	// our commitment outputs fall into this class.
-	PreschoolToKinder(*kidOutput) error
+	PreschoolToKinder(*kidOutput, uint32) error
 
 	// GraduateKinder atomically moves the kindergarten class at the
 	// provided height into the graduated status. This involves removing the
 	// kindergarten entries from both the height and channel indexes, and
 	// cleaning up the finalized kindergarten sweep txn. The height bucket
 	// will be opportunistically pruned from the height index as outputs are
 	// removed.
-	GraduateKinder(height uint32) error
+	GraduateKinder(height uint32, output *kidOutput) error
```

Comment:
godoc comments here and above should be update due to the method modifications.

Reply:
Good one, can remove a lot of comments about last grad and finalized tx.